### PR TITLE
Feature/skare 31 htmlcall

### PR DESCRIPTION
**Pull request by mccueki**
Made sure that the call button function in the flask file works with our code and has proper error handling.

### CODE REVIEW

**Review by robcas12 — APPROVED** [review]
Requirement Satisfaction: The code integrates Flask for web interaction and extends the game logic to handle player actions through web forms. Well Done!

Logic Errors: No major logic errors that I can see.

Test Coverage: The tests in htmlbutton_test.py provide coverage for the Flask app's endpoints and effectively test that the home page loads correctly and that form submissions are processed correctly. For the next sprint it might be beneficial to test for the changes made in the Call function and the pot management logic.

Style Guides: No major style guides.

**Comment by mccueki** [comment]
@robcas12 thank you!

File: source/flask_file.py [hunk 1]
```diff
@@ -34,9 +34,14 @@ def player_action():
...
-    return render_template('result.html', players=players, pot=pot.show_pot())
 
+    bet_result = Call(player, amount, pot)
+    if isinstance(bet_result, str):
+        error_message = bet_result  # Store the error message
+    else:
+        error_message = None  # No error message
+
+    return render_template('game.html', players=players, pot=pot.show_pot(), error=error_message)
+
```
Comment (erinyooey) [inline source/flask_file.py:45]: I see no apparent logic errors! Task requirement is satisfied as there is error handling and flask integration for the web through game.html :)

**Review by erinyooey — APPROVED** [review]

File: source/flask_file.py (hunk 1 above)
Comment (mccueki) [reply source/flask_file.py:45]: thanks!

**Pull request merged**